libs/gnutls/Makefile

```diff
@@ -109,19 +109,28 @@ endef
 CONFIGURE_ARGS+= \
 	--enable-shared \
 	--enable-static \
+	--disable-bash-tests \
+	--disable-doc \
+	--disable-full-test-suite \
+	--disable-gcc-warnings \
+	--disable-gtk-doc-html \
+	--disable-guile \
+	--disable-ld-version-script \
+	--disable-optional-args \
```

Comment:
what does this entail?

Reply:
No idea. I've seen and disabled it elsewhere with no negative effects.

Reply:
checking the code this is a no-op. No need to have it disabled.